### PR TITLE
feat: Add resume/suspend endpoints for CronWorkflows

### PR DESCRIPTION
**Pull request by simster7**
Closes: https://github.com/argoproj/argo/issues/4441

Signed-off-by: Simon Behar <simbeh7@gmail.com>

Checklist:

* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

### CODE REVIEW

File: pkg/apiclient/cronworkflow/cron-workflow.proto
Comment (alexec) [inline pkg/apiclient/cronworkflow/cron-workflow.proto:93]: does ResumeWorkflow use PUT?

File: pkg/apiclient/error-translating-cron-workflow-service-client.go
Comment (alexec) [inline pkg/apiclient/error-translating-cron-workflow-service-client.go:70]: what about updating `TranslateError` to return nil if `err` is nil? You could then just do this:

```golang
return workflow, grpcutil.TranslateError(err)
```

File: server/cronworkflow/cron_workflow_server.go
Comment (alexec) [inline server/cronworkflow/cron_workflow_server.go:107]: should we use patch to avoid conflicts?

File: test/e2e/argo_server_test.go
Comment (alexec) [inline test/e2e/argo_server_test.go:902]: nice

File: ui/src/app/shared/services/cron-workflow-service.ts
Comment (alexec) [inline ui/src/app/shared/services/cron-workflow-service.ts:35]: excellent

File: pkg/apiclient/cronworkflow/cron-workflow.proto
Comment (simster7) [reply pkg/apiclient/cronworkflow/cron-workflow.proto:93]: Yes it does

File: pkg/apiclient/error-translating-cron-workflow-service-client.go
Comment (simster7) [reply pkg/apiclient/error-translating-cron-workflow-service-client.go:70]: Separate PR?

File: server/cronworkflow/cron_workflow_server.go
Comment (simster7) [reply server/cronworkflow/cron_workflow_server.go:107]: Sure, but I would like to do the same for Workflow resume/suspend for consistency. Separate PR?

**Comment by alexec** [comment]
Needs code owner approval from @jessesuen

**Review by jessesuen — APPROVED** [review]

File: pkg/apiclient/error-translating-cron-workflow-service-client.go
Comment (simster7) [reply pkg/apiclient/error-translating-cron-workflow-service-client.go:70]: Opened #4465

File: server/cronworkflow/cron_workflow_server.go
Comment (simster7) [reply server/cronworkflow/cron_workflow_server.go:107]: Opened #4468

**Pull request merged**
